### ISSUE TITLE
[git] Add variable for github hostname in git_push.sh template

### ISSUE THREAD

**Issue by jhunthrop** [swagger-api/swagger-codegen#6815]
##### Description

The git_push.sh template does now allow for the target github endpoint to be changed.
A variable should be added to allow changing of this endpoint to support github enterprise usage, etc..

**Comment by wing328** [comment]
@jhunthrop yup, that's a good idea. May I know if you've time to contribute the enhancement? I can show you some good starting points.
